content/applications/finance/accounting/bank/reconciliation/cash_transactions.rst

```diff
@@ -0,0 +1,86 @@
+===================
+Cash reconciliation
+===================
+
+Overview
+========
+
+By default in Odoo, payments registered in the **cash journal** are automatically reconciled with an
+**outstanding account** and their related invoices or bills marked as *In Payment*, until they are
+reconciled with a bank statement. It is possible to :ref:`bypass <bypass>` the *In Payment* status
+for cash transactions, therefore removing the need to reconcile with a bank statement.
+
+.. important::
+   Bypassing the cash journal is recommended **only for organizations** that do not need a
+   cash report or do not wish to reconcile cash.
```

Comment:
Maybe also put a warning or a reminder in the last section where you actually explain the bypassing.